openweathermap.go

```diff
@@ -43,29 +43,52 @@ var (
 // LangCodes holds all supported languages to be used
 // inspried and sourced from @bambocher (github.com/bambocher)
 var LangCodes = map[string]string{
-	"EN":    "English",
-	"RU":    "Russian",
-	"IT":    "Italian",
-	"ES":    "Spanish",
-	"SP":    "Spanish",
-	"UK":    "Ukrainian",
-	"UA":    "Ukrainian",
-	"DE":    "German",
-	"PT":    "Portuguese",
-	"RO":    "Romanian",
-	"PL":    "Polish",
-	"FI":    "Finnish",
-	"NL":    "Dutch",
-	"FR":    "French",
-	"BG":    "Bulgarian",
-	"SV":    "Swedish",
-	"SE":    "Swedish",
-	"TR":    "Turkish",
-	"HR":    "Croatian",
-	"CA":    "Catalan",
-	"ZH_TW": "Chinese Traditional",
-	"ZH":    "Chinese Simplified",
-	"ZH_CN": "Chinese Simplified",
+	"AF":     "Afrikaans",
+	"AL":     "Albanian",
+	"AR":     "Arabic",
+	"AZ":     "Azerbaijani",
+	"BG":     "Bulgarian",
+	"CA":     "Catalan",
+	"CZ":     "Czech",
+	"DA":     "Danish",
+	"DE":     "German",
+	"EL":     "Greek",
+	"EN":     "English",
+	"EU":     "Basque",
+	"FA":     "Persian (Farsi)",
+	"FI":     "Finnish",
+	"FR":     "French",
+	"GL":     "Galician",
+	"HE":     "Hebrew",
+	"HI":     "Hindi",
+	"HR":     "Croatian",
+	"HU":     "Hungarian",
+	"ID":     "Indonesian",
+	"IT":     "Italian",
+	"JA":     "Japanese",
+	"KR":     "Korean",
+	"LA":     "Latvian",
+	"LT":     "Lithuanian",
+	"MK":     "Macedonian",
+	"NL":     "Dutch",
+	"NO":     "Norwegian",
+	"PL":     "Polish",
+	"PT":     "Portuguese",
+	"PT_BR":  "Português Brasil",
```

Comment:
These double entries need to be individual entries like they originally were.

Reply:
sorry, what do you mean?

Reply:
The key "SP, ES" was "SP", and "ES". The should go back to that.

Reply:
I've fixed it.